components/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC, PropsWithChildren } from 'react';
+
+interface Props {
+  enabled: boolean;
+}
+
+const Button: FC<PropsWithChildren<Props>> = ({ children, enabled }) => {
+  return (
+    <button
+      disabled={!enabled}
+      className="bg-ocf-yellow disabled:bg-ocf-gray disabled:dark:bg-ocf-gray shadow h-full w-full text-center rounded-md font-bold text-xl uppercase"
```

Comment:
We can probably set a specific height on this button component—perhaps `h-14` could work? I don't see the height of this component changing all that much, if at all

Reply:
also maybe consider a max width—currently the button looks a big wonky as screen sizes get larger